src/main/java/com/plugish/woominecraft/WooMinecraft.java

```diff
@@ -69,13 +71,43 @@ public void onEnable() {
 
 		// Log when plugin is fully enabled ( setup complete ).
 		getLogger().info( this.getLang( "log.enabled" ) );
-		//check bungeecord mode/offline mode
-		if (!Bukkit.getOnlineMode() && !Bukkit.spigot().getConfig().getBoolean("settings.bungeecord")) {
-			getLogger().severe(String.valueOf(Bukkit.spigot().getConfig().getBoolean("settings.bungeecord")));
-			getLogger().severe("WooMinecraft doesn't support offLine mode");
-			Bukkit.getPluginManager().disablePlugin(this);
+		//check bungeecord/velocity mode/offline mode
+		try {
+			getLogger().info(Bukkit.getServer().getName());
+			//check if the server is spigot/bukkit
+			if (Bukkit.getServer().getName().equals("CraftBukkit"))
+				if (!Bukkit.getOnlineMode() && !Bukkit.spigot().getConfig().getBoolean("settings.bungeecord")) {
+					getLogger().severe("BungeeCord: "+Bukkit.spigot().getConfig().getBoolean("settings.bungeecord"));

```

Comment:
Given how many `getLogger.xxx` calls we have here, and the depth of the nesting, I'd either like to see this nesting simplified ( by creating separate methods ) or maybe create a logging class entirely and calling it here. Perhaps this is the web developer in me but I'm not used to seeing such deeply nested IF statements.